apps/web/vibes/soul/sections/product-detail/index.tsx

```diff
@@ -97,12 +99,20 @@ export function ProductDetail<F extends Field>({
                     }
                   </Stream>
 
-                  <Stream fallback={<ProductDetailFormSkeleton />} value={streamableFields}>
-                    {(fields) => (
+                  <Stream
+                    fallback={<ProductDetailFormSkeleton />}
+                    value={Promise.all([
+                      streamableFields,
+                      streamableSubmitButtonLabel,
+                      streamableSubmitButtonDisabled,
+                    ])}
+                  >
+                    {([fields, submitButtonLabel, submitButtonDisabled]) => (
                       <ProductDetailForm
                         action={action}
-                        ctaLabel={ctaLabel}
```

Comment:
Seeing this diff makes me wonder, should we name the props `ctaLabel` and `ctaDisabled`? Or does "submit" make more sense in this context than "call to action"?

Reply:
Either way is fine by me.